devel/libfilezilla/Portfile

```diff
@@ -28,6 +28,11 @@ checksums           rmd160  c8dba10cb670364777b29f797f9852f00708ffa6 \
                     sha256  c5f660c4b339ed6a9e89ec0bafd5d39ba53cb3a3b52051bc571bb74e03ef8f7a \
                     size    557320
 
+patchfiles          patch-lib-process.cpp.diff
```

Comment:
could you add a comment here explains the patch.
Perfect if you can include a link to an upstream issue for it, so we can follow when the patch is no longer required.

Reply:
perfect, thanks. Lets just now wait for the ci tests to complete, just to be sure still OK here.